packages/sdk-react/rollup.config.mjs

```diff
@@ -1,47 +1,54 @@
-import typescript from "@rollup/plugin-typescript";
-import nodeExternals from "rollup-plugin-node-externals";
-import path from "node:path";
-import postcss from "rollup-plugin-postcss";
-import preserveDirectives from "rollup-preserve-directives";
-import url from "@rollup/plugin-url";
-import alias from "@rollup/plugin-alias";
-import copy from "rollup-plugin-copy";
+import typescript from '@rollup/plugin-typescript';
```

Comment:
would expect these to be double quotes

Reply:
from all the package READMES it shows single quotes actually. It seems like we were incorrect before